build.gradle

```diff
@@ -47,6 +47,7 @@ task downloadIntelDeflater(type: Download) {
     //IntelDeflater is not packages with the rest of HTSJDK so we download it by hand
     src 'https://github.com/samtools/htsjdk/raw/2.1.1/lib/jni/libIntelDeflater.so'
     dest 'build/'
+    overwrite false
```

Comment:
It doesn't work the way I had hoped.  I've opened an issue with the download plugin authors to see if there is a way to achieve my desired behavior. https://github.com/michel-kraemer/gradle-download-task/issues/42

Reverting.